vizier/datastore/mimir/base.py

```diff
@@ -185,23 +185,10 @@ def get_select_query(table_name, columns=None):
     """
     if not columns is None:
         col_list = ','.join([col.name_in_rdb for col in columns])
-        return 'SELECT ' + ROW_ID + ',' + col_list + ' FROM ' + table_name
+        return 'SELECT ' + col_list + ' FROM ' + table_name
```

Comment:
does this need to be SQL?  Could we expose a mimir API to do simple select/project/limit queries instead?  Less parsing = less headache.